jekyll/_sass/_instantsearch.scss

```diff
@@ -0,0 +1,74 @@
+.base-input-style {
+  border-radius: 0;
+  padding: 1.25em 1em 1em 2.8em;
```

Comment:
Do we use `em` units elsewhere in this project? I might be the only one, but I get confused each time I see it 🙈 If possible, I think `px` would be clearer